src/UIComponents/AddFeatureButton.jsx

```diff
@@ -0,0 +1,43 @@
+/* React component to handle selecting columns as features. */
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { styles } from "../constants";
+import { addSelectedFeature } from "../redux";
+
+
+class AddFeatureButton extends Component {
+  static propTypes = {
+    column: PropTypes.string,
+    addSelectedFeature: PropTypes.func.isRequired
+  };
+
+  addFeature = e => {
```

Comment:
nit: the next file uses `event`.  Should we consistently use `e` or `event` across the codebase?

Reply:
sure. I'll go with "event" since it's easier to search unless you have a preference.

Reply:
Sounds good.